boards/nucleo144-f413/board.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016 Inria
```

Comment:
As I didn't change the code from you PR, I didn't think about the copyright. Addressed